.github/workflows/close-stale.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 6 * * 1'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          days-before-stale: 60
```

Comment:
You are right in that it's 60+14 days before an issue(or PR) is closed.

It takes 60 days to **active issue** -> **stale issue**.
It takes 14 days to **stale issue** -> **closed issue**.

So "stale" period is kind of a "heads-up, we're about to close it"-thing and only starts after 60 days so in that sense saying "This issue is closed because it has been **stale** for 74 days" is not correct.